packages/kas/src/__tests__/comparing_test.js

```diff
@@ -11,6 +12,7 @@ expect.extend({
 
         const actual = KAS.compare(inputExpr, expectedExpr, {form: false});
 
+        // $FlowFixMe[object-this-reference]
         if (this.isNot) {
```

Comment:
Be cool if we could use the `this` type annotation for the functions... but I checked and the `extend` function isn't very well-typed so that's not possible. 

https://flow.org/en/docs/types/functions/#toc-function-this